src/frontend/src/flows/manage/deviceSettings.ts

```diff
@@ -116,12 +114,10 @@ const deviceConnection = async (
 
 /* Remove the device and return */
 const deleteDevice = async ({
-  userNumber,
```

Comment:
Some logic was there only for protected phrases; by not allowing phrases here anymore things can be simplified a bit